tests/test_deployment.py

```diff
@@ -495,12 +495,12 @@ def test_scales_dynos(self, dsss, heroku_mock, active_config):
         )
 
     def test_scales_different_dynos(self, dsss, heroku_mock, active_config):
-        active_config.set("dyno_type", u"ignored")
```

Comment:
I think these changes are incorrect. The variable is defined as using `six.text_type`, so it should have the `u` prefix for Python2 compatibility. Besides, this is a strange thing to include in the version bump commit.

Reply:
I think Black made this change because there's now a `from __future__ import unicode_literals` at the top of the file.